src/account/AccountLoupe.sol

```diff
@@ -38,7 +39,7 @@ abstract contract AccountLoupe is IAccountLoupe {
             config.plugin = _storage.selectorData[selector].plugin;
         }
 
-        config.validationFunction = _storage.selectorData[selector].validation;
+        config.defaultValidationFunction = toFunctionReference(_storage.selectorData[selector].validations.at(0));
```

Comment:
(small nit) can consider renaming it to `storedValidationFunction` + describe it in the spec as `if functions are called directly on an account, the stored validation function would be the one that's used`